app/views/rails_admin/main/export.html.erb

```diff
@@ -1,6 +1,6 @@
 <% params = request.params.except(:action, :controller, :utf8, :page, :per_page, :format, :authenticity_token) %>
 <% visible_fields = @model_config.export.with(view: self, object: @abstract_model.model.new, controller: self.controller).visible_fields %>
-<%= form_tag export_path(params.merge(all: true)), method: 'post', class: 'form-horizontal' do %>
+<%= form_tag export_path(params.merge(all: true)), method: 'post', class: "main" do %>
   <input name="send_data" type="hidden" value="true">/</input>
```

Comment:
(Not related to the bootstrap changes but noticed in review) this is rendering the `/` to the page, seems to have come from https://github.com/railsadminteam/rails_admin/commit/aa6ef95d6c1cdd2daa81b725be7569e404251a77